tests/phpunit/tests/cron.php

```diff
@@ -271,8 +271,12 @@ function test_duplicate_event() {
 
 		// First one works.
 		$this->assertTrue( wp_schedule_single_event( $ts1, $hook, $args ) );
-		// Second one is ignored.
+
+		// Subsequent ones are ignored.
 		$this->assertFalse( wp_schedule_single_event( $ts2, $hook, $args ) );
+		$subsequent = wp_schedule_single_event( $ts2, $hook, $args, true );
+		$this->assertWPError( $subsequent );
```

Comment:
That's actually the reason for this assertion. If this assertion is not there and `$subsequent` isn't a `WP_Error` then the next assertion will cause error instead of a failure.